runner/__init__.py

```diff
@@ -192,6 +203,9 @@ def make_argument_parser():
     parser.add_argument("-n", "--times", dest="times", type=int, help="run this many times (default is forever)")
     parser.add_argument("-H", "--halt-after", dest="halt_after", action="store_const", const=True,
                         help="Call the halt task after runner finishes (never called if -n is not set).")
+    parser.add_argument("--syslog", dest="syslog", action="store_const", const=True, help="send messages to syslog")
```

Comment:
I believe this should be action="store_true". Also, dest=... is redundant if if matches the --value.